aries_cloudagent/protocols/connections/manager.py

```diff
@@ -838,13 +840,19 @@ def context(self) -> InjectionContext:
                     True,
                 )
                 routing_keys.append(rk)
-                svc_endpoint = service.endpoint
+                svc_endpoint = [service.endpoint]
```

Comment:
Should be `svc_endpoints` according to LGTM

Reply:
good catch. fixed.